backend/e4h-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -2437,3 +2440,516 @@ def find_col(partial):
+            os.unlink(input_temp_file.name)
+        if output_temp_file and os.path.exists(output_temp_file.name):
+            os.unlink(output_temp_file.name)
+        raise HTTPException(status_code=500, detail=f"An unexpected error occurred: {str(e)}")
+
+
+@router.post('/amcConfigurationBulkIngest',
+             summary='Bulk ingest AMC configuration template data',
+             response_description="Returns processed Excel file with AMC configuration creation results")
+async def bulk_ingest_amc_configurations(
+        background_tasks: BackgroundTasks,
+        amc_file: UploadFile = File(..., description="Excel file containing AMC configuration data"),
```

Ruff (0.14.5)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)